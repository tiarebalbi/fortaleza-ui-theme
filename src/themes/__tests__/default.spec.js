import * as React from 'react';
import { palette } from 'styled-theme';
import { mount } from 'enzyme';
import styled from 'styled-components';
import theme from '../default';
import toJson from 'enzyme-to-json';

const Main = styled.section`
  padding: 4em;
  background: ${palette(0)({ theme: theme, palette: 'primary' })};
`;

const setup = () => {
  return <Main />;
};

describe('default/theme', () => {
  const wrapper = mount(setup());

  it('should match style', () => {
    expect(wrapper).toExist();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
