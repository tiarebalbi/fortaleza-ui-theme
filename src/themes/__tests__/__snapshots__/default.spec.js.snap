// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default/theme should match style 1`] = `
<styled.section>
  <section
    className="sc-bdVaJa eIkkgE"
  />
</styled.section>
`;
